maintenance/Maintenance.rdf

```diff
@@ -242,303 +268,445 @@
+                        <owl:someValuesFrom rdf:resource="https://purl.industrialontologies.org/ontology/core/Core/MaintainableMaterialItem"/>
                     </owl:Restriction>
                 </owl:intersectionOf>
             </owl:Class>
         </owl:equivalentClass>
-        <rdfs:subClassOf rdf:resource="https://purl.industrialontologies.org/ontology/core/Core/Person" />
-        <rdfs:label>Qualified Person</rdfs:label>
-        <iof-av:firstOrderLogicDefinition>Person(x) ∧ ∃y(MaintenanceActivity(y) ∧ partcipatesInAtSomeTime(x,y))  ∧ ∃z(Qualification(z) ∧ satisfies(x,z))  ↔ QualifiedPerson(x)</iof-av:firstOrderLogicDefinition>
-        <iof-av:naturalLanguageDefinition>A person qualified to perform a specified process or task</iof-av:naturalLanguageDefinition>
-        <iof-av:semiFormalNaturalLanguageDefinition>&apos;qualified person&apos;: every  instance of ‘qualified person’ is defined as exactly an instance of ‘person’ that ‘participates in at some time’ some ‘maintenance activity’ and &apos;satifies&apos; some &apos;qualification&apos;</iof-av:semiFormalNaturalLanguageDefinition>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/BFO_0000034"/>
+        <rdfs:label>required function</rdfs:label>
+        <rdfs:seeAlso>ISO81346 for examples of functions</rdfs:seeAlso>
+        <skos:example>to sense, to store, to process information, to control</skos:example>
+        <iof-av:adaptedFrom>The natural language definition is adapted from EN:13306</iof-av:adaptedFrom>
+        <iof-av:explanatoryNote>the function can be a combination of functions</iof-av:explanatoryNote>
+        <iof-av:firstOrderLogicDefinition>Function(x) ∧ ∃y,z(MaintainableMaterialItem(y) ∧ FunctioningProcess(z) ∧ participatesInAtSomeTime(y,z) ∧ functionOf(x,y)) ↔ RequiredFunction(x)</iof-av:firstOrderLogicDefinition>
         <iof-av:isPrimitive rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">false</iof-av:isPrimitive>
+        <iof-av:maturity>Release</iof-av:maturity>
+        <iof-av:naturalLanguageDefinition>function of a maintainable item which is considered necessary to fulfil a process requirement</iof-av:naturalLanguageDefinition>
+        <iof-av:semiFormalNaturalLanguageDefinition>required function&apos;: every instance of ‘required function’ is defined as an instance of ‘function’ that is the &apos;function of&apos; some &apos;maintainable material item&apos; that &apos;participates in at some time&apos; some &apos;functioning process&apos;</iof-av:semiFormalNaturalLanguageDefinition>
+        <iof-av:synonym>primary function</iof-av:synonym>
     </owl:Class>
+    
 
 
     <!-- https://dev.industrialontologies.org/ontology/maintenance/MaintenanceReferenceOntology/SupportingActivity -->
 
     <owl:Class rdf:about="https://dev.industrialontologies.org/ontology/maintenance/MaintenanceReferenceOntology/SupportingActivity">
```

Comment:
Need to add the examples and explanatory note. My bad for not highlighting properly in the spreadsheet